tgui/packages/tgui-panel/telemetry.js

```diff
@@ -71,7 +71,7 @@ export const telemetryMiddleware = (store) => {
         // Save telemetry
         if (telemetryMutated) {
           logger.debug('saving telemetry to storage', telemetry);
-          storage.set('telemetry', telemetry);
+          storage.set_noconfig('telemetry', telemetry);
```

Comment:
This means basically everyone's existing telemetry will be cleared. Ideally it should use the default (old) save path that all servers use for telemetry so old and new tgchat instances will share telemetry data.

To explain, telemetry is how admins track past connections from users to other servers and the current server. CID, IPs, CKEYs, etc and is very important for evasion detection. The perk is the localStorage is shared between all servers, so your average Evader McEvadeChamp will go strutting between servers with the same telemetry data, racking up banned ckeys in their telemetry, admins can check the centcom database to see what they've been up to in the past and make judgements based on that of if they're a griefer or not.

Changing this to anything other than the default means any codebase without this PR will not share telemetry with any codebase with this PR, making the amount of telemetry sent between the two much lower and thus ban evasion much easier

Reply:
`set_noconfig()` function uses "constant(=old)" save path, so it's addressed already.